modal-popup.css

```diff
@@ -0,0 +1,204 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+body {
+  background: #eee;
+}
+
+/* PROJECT POPUP STARTS */
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  min-height: 100vh;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.8);
+  backdrop-filter: blur(4px);
+  display: none;
+  align-items: center;
+  justify-content: center;
+  z-index: 7;
+}
+
+.modal-container {
+  width: 86%;
+  margin: 0 auto;
+}
+
+#modal .popup {
+  padding: 0.5rem;
+  text-align: center;
+  background: #fff;
+  position: relative;
+  width: 50%;
+  margin: 0 auto;
+  border-radius: 0.5rem;
+  z-index: 5;
+}
```

Comment:
- [ ] You have done a very great job to ensure that the modal popup is functioning as required in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). I have noticed the width of the modal is not matching the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1143&t=ZdpN3Y16S4Q97h99-0). You should ensure the popup covers at least `80%` of the screen size and at least `9%` gap between the modal and body borders. 
- [ ] In addition, kindly ensure that the contents of the modal are all wrapped within. I, therefore, suggest considering having the vertical scroll. This will help us to match the Figma design in such a way that all contents can be easily accessed and also the bottom gap is established. This improves the user experience of your portfolio.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/232011934-21641d12-e9ea-46dd-b5fd-f12f68fc041d.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/232011621-a89458f7-9a11-4817-9677-176848f17522.png)

Reply:
This is very helpful feedback. I've implemented all the suggestions you made. Thanks.